tutorials/13_diagnosis_sbc.ipynb

```diff
@@ -0,0 +1,1897 @@
+      "text/plain": [
+       "<Figure size 576x360 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from sbi.analysis.plot import sbc_rank_plot\n",
+    "\n",
+    "f, ax = sbc_rank_plot(ranks, 1_000, plot_type = \"hist\", num_bins=30)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "The two plots show only visualize the histogram across the `ranks` (here depicted in red). Highlighted in grey is the 95% confidence interval of a uniform distribution given the number of samples provided. In plain english, the majority of red bars should reach the grey area of the confidence interval. \n",
+    "\n",
+    "We also observe, that the entries fluctuate to some degree. This can be considered a hint that `sbc` should be conducted with a lot more samples than 1000."
```

Comment:
I think there can be other reasons, e.g., the number of bins, it changes the sensitivity of the visual uniformity check. There is a rule of thumb for choosing number of the bins `B` and the number of SBC samples `N`, in order to obtain a good trade-off: `N / B ~ 20`. You could mention this here, or drop this sentence entirely

Reply:
26b306b